File: Microsoft.CodeAnalysis.VisualBasic/Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax/ScannerState.cs
namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
{
	internal enum ScannerState
	{
		VB,
		VBAllowLeadingMultilineTrivia,
		Misc,
		DocType,
		Element,
		EndElement,
		SingleQuotedString,
		SmartSingleQuotedString,
		QuotedString,
		SmartQuotedString,
		UnQuotedString,
		Content,
		CData,
		StartProcessingInstruction,
		ProcessingInstruction,
		Comment,
		InterpolatedStringPunctuation,
		InterpolatedStringContent,
		InterpolatedStringFormatString
	}
}

File: Microsoft.CodeAnalysis.VisualBasic/Microsoft.CodeAnalysis/VisualBasic/Syntax/InternalSyntax/ScannerState.vb
Imports System

Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
	Friend Enum ScannerState
		VB
		VBAllowLeadingMultilineTrivia
		Misc
		DocType
		Element
		EndElement
		SingleQuotedString
		SmartSingleQuotedString
		QuotedString
		SmartQuotedString
		UnQuotedString
		Content
		CData
		StartProcessingInstruction
		ProcessingInstruction
		Comment
		InterpolatedStringPunctuation
		InterpolatedStringContent
		InterpolatedStringFormatString
	End Enum
End Namespace